next.config.js

```diff
@@ -4,15 +4,15 @@ const path = require('path');
  * @type {import('next').NextConfig}
  */
 const nextConfig = {
-  output: 'export',
+  output: 'standalone',
```

Comment:
Looks like this is breaking the CI checks since we're no longer producing an `out` directory containing the static files. 

Should we leave it as `export` unless building within a Docker image? Or update CI to handle a `standalone` build?

<img width="1160" alt="image" src="https://github.com/user-attachments/assets/4377c6cd-0446-4679-883d-4a51cff43d0c">